### ISSUE TITLE
Standardize phone/fax number formats

### ISSUE THREAD

**Issue by cthomasONRR** [ONRR/onrr.gov-site#1926]
In the new CMS, all phone numbers should be in ###-###-#### format and NOT (###) ###-####. Sid says the ###-###-#### format is easier for screen readers.

**Comment by cthomasONRR** [comment]
Webpages to update:

- [ ] https://dev-onrr-frontend.app.cloud.gov/getting-started/new-reporter-checklist?tabs=oil-gas-production
- [ ] https://dev-onrr-frontend.app.cloud.gov/getting-started/new-reporter-checklist?tabs=oil-gas-revenue
- [ ] https://dev-onrr-frontend.app.cloud.gov/getting-started/new-reporter-checklist?tabs=solid-minerals
- [ ] https://dev-onrr-frontend.app.cloud.gov/getting-started/new-reporter-checklist?tabs=geothermal
- [ ] https://dev-onrr-frontend.app.cloud.gov/getting-started/system-access?tabs=setting-up-access
- [ ] https://dev-onrr-frontend.app.cloud.gov/getting-started/system-access?tabs=troubleshooting
- [ ] https://dev-onrr-frontend.app.cloud.gov/reporting/production?tabs=new-reporter-checklist
- [ ] https://dev-onrr-frontend.app.cloud.gov/reporting/production?tabs=troubleshooting
- [ ] https://dev-onrr-frontend.app.cloud.gov/reporting/revenue?tabs=new-reporter-checklist
- [ ] https://dev-onrr-frontend.app.cloud.gov/reporting/revenue?tabs=troubleshooting
- [ ] https://dev-onrr-frontend.app.cloud.gov/reporting/solids?tabs=new-reporter-checklist
- [ ] https://dev-onrr-frontend.app.cloud.gov/reporting/geothermal?tabs=new-reporter-checklist
- [ ] https://dev-onrr-frontend.app.cloud.gov/reporting/geothermal?tabs=troubleshooting
- [ ] https://dev-onrr-frontend.app.cloud.gov/references/handbooks/minerals-revenue-reporter-handbook?tabs=contact
- [ ] https://dev-onrr-frontend.app.cloud.gov/references/handbooks/minerals-production-reporter-handbook?tabs=contact
- [ ] https://dev-onrr-frontend.app.cloud.gov/references/handbooks/geothermal-class-1?tabs=contact
- [ ] https://dev-onrr-frontend.app.cloud.gov/references/handbooks/geothermal-class-2-and-3?tabs=contact
- [ ] https://dev-onrr-frontend.app.cloud.gov/paying/unpaid-debt
- [ ] https://dev-onrr-frontend.app.cloud.gov/compliance-enforcement/cooperative?tabs=ipa-fellowship-program
- [ ] https://dev-onrr-frontend.app.cloud.gov/indian/assistance?tabs=indian-outreach,overview
- [ ] https://dev-onrr-frontend.app.cloud.gov/indian/assistance?tabs=federal-indian-minerals-office
- [ ] https://dev-onrr-frontend.app.cloud.gov/indian/assistance?tabs=tribal-assistance
- [ ] https://dev-onrr-frontend.app.cloud.gov/indian/assistance?tabs=indian-energy-service-center,about
- [ ] https://dev-onrr-frontend.app.cloud.gov/indian/assistance?tabs=indian-energy-service-center,referral-information
- [ ] https://dev-onrr-frontend.app.cloud.gov/indian/assistance?tabs=contacts,regional-iimo-contacts
- [ ] https://dev-onrr-frontend.app.cloud.gov/indian/indian-references?tabs=contacts,regional-iimo-contacts
- [ ] https://dev-onrr-frontend.app.cloud.gov/about/public-affairs?tabs=contact-social-media
- [ ] https://dev-onrr-frontend.app.cloud.gov/about/office-locations

**Comment by cthomasONRR** [comment]
Cross-referencing my list with #1854

**Comment by cthomasONRR** [comment]
@Baymatt @mcharg This is ready for review. Please review the webpages listed above and make sure all phone/fax numbers are in the ###-###-#### format. This excludes the actual contact card collections, we are waiting for Jerome to upload a spreadsheet of those as the update.

**Comment by mcharg** [comment]
Webpages to review:

- [ ] https://dev-onrr-frontend.app.cloud.gov/getting-started/new-reporter-checklist?tabs=oil-gas-production
- [x] https://dev-onrr-frontend.app.cloud.gov/getting-started/new-reporter-checklist?tabs=oil-gas-revenue
- [x] https://dev-onrr-frontend.app.cloud.gov/getting-started/new-reporter-checklist?tabs=solid-minerals
- [x] https://dev-onrr-frontend.app.cloud.gov/getting-started/new-reporter-checklist?tabs=geothermal
- [x] https://dev-onrr-frontend.app.cloud.gov/getting-started/system-access?tabs=setting-up-access
- [x] https://dev-onrr-frontend.app.cloud.gov/getting-started/system-access?tabs=troubleshooting
- [x] https://dev-onrr-frontend.app.cloud.gov/reporting/production?tabs=new-reporter-checklist
- [x] https://dev-onrr-frontend.app.cloud.gov/reporting/production?tabs=troubleshooting
- [x] https://dev-onrr-frontend.app.cloud.gov/reporting/revenue?tabs=new-reporter-checklist
- [x] https://dev-onrr-frontend.app.cloud.gov/reporting/revenue?tabs=troubleshooting
- [x] https://dev-onrr-frontend.app.cloud.gov/reporting/solids?tabs=new-reporter-checklist
- [x] https://dev-onrr-frontend.app.cloud.gov/reporting/geothermal?tabs=new-reporter-checklist
- [x] https://dev-onrr-frontend.app.cloud.gov/reporting/geothermal?tabs=troubleshooting
- [x] https://dev-onrr-frontend.app.cloud.gov/references/handbooks/minerals-revenue-reporter-handbook?tabs=contact
- [x] https://dev-onrr-frontend.app.cloud.gov/references/handbooks/minerals-production-reporter-handbook?tabs=contact
- [x] https://dev-onrr-frontend.app.cloud.gov/references/handbooks/geothermal-class-1?tabs=contact
- [x] https://dev-onrr-frontend.app.cloud.gov/references/handbooks/geothermal-class-2-and-3?tabs=contact
- [x] https://dev-onrr-frontend.app.cloud.gov/paying/unpaid-debt
- [x] https://dev-onrr-frontend.app.cloud.gov/compliance-enforcement/cooperative?tabs=ipa-fellowship-program
- [x] https://dev-onrr-frontend.app.cloud.gov/indian/assistance?tabs=indian-outreach,overview
- [x] https://dev-onrr-frontend.app.cloud.gov/indian/assistance?tabs=federal-indian-minerals-office
- [x] https://dev-onrr-frontend.app.cloud.gov/indian/assistance?tabs=tribal-assistance
- [x] https://dev-onrr-frontend.app.cloud.gov/indian/assistance?tabs=indian-energy-service-center,about
- [x] https://dev-onrr-frontend.app.cloud.gov/indian/assistance?tabs=indian-energy-service-center,referral-information
- [x] https://dev-onrr-frontend.app.cloud.gov/indian/assistance?tabs=contacts,regional-iimo-contacts
- [x] https://dev-onrr-frontend.app.cloud.gov/indian/indian-references?tabs=contacts,regional-iimo-contacts
- [x] https://dev-onrr-frontend.app.cloud.gov/about/public-affairs?tabs=contact-social-media
- [x] https://dev-onrr-frontend.app.cloud.gov/about/office-locations

**Comment by cthomasONRR** [comment]
@mcharg Great! Closing this issue now.
